app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultationType.kt

```diff
@@ -0,0 +1,45 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import androidx.room.TypeConverter
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import org.simple.clinic.util.room.SafeEnumTypeAdapter
+
+sealed class TeleconsultationType {
+
+  object Audio : TeleconsultationType()
+
+  object Video : TeleconsultationType()
+
+  object Message : TeleconsultationType()
```

Comment:
Can we override the `.toString()` method on each of the `object` types so that they behave as if they were enums instead of sealed class values?